### PR TITLE
Model card for dkleczek/bert-base-polish-uncased-v1

### PR DESCRIPTION
**Pull request by kldarek**
Model card for dkleczek/bert-base-polish-uncased-v1

### CODE REVIEW

**Comment by julien-c** [comment]
Thanks for sharing => [model page](https://huggingface.co/dkleczek/bert-base-polish-uncased-v1)

**Comment by julien-c** [comment]
will fix images + lowercase the language tag in the next commit

File: model_cards/dkleczek/bert-base-polish-uncased-v1/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    last_hidden_states = bert_model(input_ids)[0]
+more_words = np.argsort(np.asarray(last_hidden_states[0,masked_token,:]))[-4:]
+print(more_words)
+
+# Output: 
+# poeta
+# bohaterem
+# człowiekiem
+# pisarzem
+```
```
Comment (julien-c) [inline model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: Here, you could alternatively use our `FillMaskPipeline` which does the above in a one-liner

File: model_cards/dkleczek/bert-base-polish-uncased-v1/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Evaluation
+I'd love to get some help from the Polish NLP community here! If you feel like evaluating Polbert on some benchmark tasks, it would be great if you can share the results. 
+
+So far, I've compared the performance of Polbert vs Multilingual BERT on PolEmo 2.0 sentiment classification, here are the results. These results are are produced with a linear classification layer on top of pooled output, trained for 10 epochs with learning rate 3e-5. The checkpoint with the lowest loss on validation set is evaluated on the test set. 
+
+| PolEmo 2.0 Sentiment Classifcation | Test Accuracy | 
+| ------------- |--------------:|
+| Multilingual BERT | 0.78 |
+| Polbert | 0.85 |
```
Comment (julien-c) [inline model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: this is really cool @kldarek !

Comment (julien-c) [reply model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: is the sentiment classification head inside those weights too?

File: model_cards/dkleczek/bert-base-polish-uncased-v1/README.md (hunk 1 above)
Comment (kldarek) [reply model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: I will look into this now - just learned about the Pipeline yesterday!

File: model_cards/dkleczek/bert-base-polish-uncased-v1/README.md (hunk 2 above)
Comment (kldarek) [reply model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: Thanks! No, I only uploaded the pretraining weights. I finetuned the whole model for sentiment classification. If you recommend to include custom heads I can look into that :)

Comment (julien-c) [reply model_cards/dkleczek/bert-base-polish-uncased-v1/README.md]: Yeah; you could upload as a separate model, and then everyone can use it in a `"sentiment-analysis"` `TextClassificationPipeline`
